### PR TITLE
Add job to scan flux-operator in the Vulnerability scan workflow

### LINKED ISSUE #132: Add flux-operator to the cve scan workflow
We should scan the latest version of flux-operator from the `ghcr.io/controlplaneio-fluxcd/flux-operator` repo.

### PR DESCRIPTION
**Pull request by matheuscscp**
Fixes #132 

Tested here: https://github.com/controlplaneio-fluxcd/distribution/actions/runs/12319974501/job/34388241334

### CODE REVIEW

**Comment by matheuscscp** [comment]
Tested here: https://github.com/controlplaneio-fluxcd/distribution/actions/runs/12319974501/job/34388241334

File: .github/workflows/scan-distribution.yaml [hunk 1]
```diff
@@ -24,3 +24,43 @@ jobs:
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        VERSION=$(gh release view --repo controlplaneio-fluxcd/flux-operator --json tagName -q .tagName)
+        echo version=${VERSION} >> $GITHUB_OUTPUT
+    - name: Export image
+      id: image
+      run: echo fo=${REGISTRY}:${VERSION} >> $GITHUB_OUTPUT
+      env:
+        REGISTRY: ghcr.io/controlplaneio-fluxcd/flux-operator
```
Comment (matheuscscp) [inline .github/workflows/scan-distribution.yaml:44]: https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

File: .github/workflows/scan-distribution.yaml [hunk 2]
```diff
@@ -24,3 +24,43 @@ jobs:
...
+        VERSION: ${{ steps.release.outputs.version }}
```
Comment (matheuscscp) [inline .github/workflows/scan-distribution.yaml:45]: https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @matheuscscp 🏅

**Pull request merged**